frontend/src/lib/components/Editor.svelte

```diff
@@ -45,7 +100,12 @@
 		</svg>
 	</button>
 	<!-- Save -->
-	<button on:click={saveChangesHandler} class="publish" title="Publish Changes">
+	<button 
+	on:click={async () => {
+		showCommitModal = true;
+		await tick();
```

Comment:
I don't believe the `await tick();` here has any effect because there's no code after it that involves state that would have changed during that tick